src/View/html/FormSettings/add_edit.php

```diff
@@ -18,6 +18,11 @@
 
 global $wp_settings_fields;
 
+$additional_form_class = '';
+
+if ( version_compare( '2.6', GFCommon::$version, '>=' ) ) {
```

Comment:
I think we need to check `2.6-beta-1` rather than just `2.6` for this to work with the beta correctly. 

Also, this logic should be moved to the Controller or Model (depending who calls `add_edit`) and then injected into the view.

Reply:
We should probably use a class name that isn't so generic either like `gfpdf-gf-2-6`